teams/hubturbo.md

```diff
@@ -30,7 +30,7 @@ permalink: /teams/hubturbo/
 
  - [@xinan](https://github.com/xinan)
    ([Liu Xinan](http://xinan.io))
-   - Technology Expertise:
+   - Technology Expertise: Performance Optimization
```

Comment:
Keep in mind that expertise doesn't need to be related to your major/minor projects. It can be anything you want to claim expertise of, when you apply for jobs for example.